assets/app/view/game/actionable.rb

```diff
@@ -68,6 +69,7 @@ def process_action(action)
 
         store(:tile_selector, nil, skip: true)
         store(:game, game)
+        store(:fullgame, game)
```

Comment:
double render

Reply:
ahh, i'd assumed it would swallow the second render